### ISSUE TITLE
App freeze when app has no interent connection

### ISSUE THREAD

**Issue by sushilbalami** [bdlukaa/native_admob_flutter#6]
When there are no internet connection app freezes while loading ads on the app
I put the below code on the bottom navigation bar so when there is no internet connection app freezes.
while loading native ads
```
NativeAd(
            controller: _nativAdController,
            height: 60,
            buildLayout: adBannerLayoutBuilder,
            loading: Text('Loading'),
            error: Text('Error'),
            icon: AdImageView(padding: EdgeInsets.only(left: 6)),
            headline: AdTextView(
              style: TextStyle(color: Colors.black),
              maxLines: 1,
            ),
            advertiser: AdTextView(style: TextStyle(color: Colors.black)),
            body: AdTextView(style: TextStyle(color: Colors.black)),
            media: AdMediaView(height: 70, width: 120),
            button: AdButtonView(
              margin: EdgeInsets.only(left: 6, right: 6),
              textStyle: TextStyle(color: Colors.green, fontSize: 14),
            ),
          ),
```
Sometimes on internet connection too
![Screenshot_2021-02-14-14-01-52-277_com.miui.bugreport.jpg](https://user-images.githubusercontent.com/33893625/107878415-9aa02d00-6efa-11eb-8527-a2d597036165.jpg)
![Screenshot_2021-02-14-14-01-54-387_com.miui.bugreport.jpg](https://user-images.githubusercontent.com/33893625/107878416-9c69f080-6efa-11eb-962d-7138636a50ee.jpg)
![Screenshot_2021-02-14-14-02-18-256_com.miui.bugreport.jpg](https://user-images.githubusercontent.com/33893625/107878417-9e33b400-6efa-11eb-84fb-4919cab82139.jpg)

**Comment by bdlukaa** [comment]
Thanks for reporting this error. I just released a new version ([v0.8.2](https://pub.dev/packages/native_admob_flutter)) with the fix for it. If you still see the error with that version, please open a new issue with more details.

**Comment by sushilbalami** [comment]
ok
